.autorc

```diff
@@ -3,5 +3,13 @@
     "baseBranch": "master",
     "author": "auto <auto@nil>",
     "noVersionPrefix": true,
-    "plugins": ["git-tag"]
+    "plugins": [
+        "git-tag",
+        [
+            "exec",
+            {
+                "beforeCommitChangelog": "awk 'NR==1 {print \"__version__ = \\\"\" $2 \"\\\"\"}' CHANGELOG.md > datalad_crawler/version.py && git add datalad_crawler/version.py"
```

Comment:
wouldn't it be reasonably easy to add a call to `python setup.py --name` to avoid hardcoding of the package name here?

Reply:
ref: https://github.com/datalad/datalad-crawler/pull/96/files#r614408339

Reply:
If the goal is to make the code copy-and-pastable between projects, that won't work in general, as `setup.py --name` returns the project name, rather than the name of the Python package.

Reply:
for current target use cases it seems we are fine
```
$> grep name.*datalad datalad-{container,crawler}/setup.py
datalad-container/setup.py:    name="datalad_container",
datalad-crawler/setup.py:    name="datalad_crawler",
```
and it would work at least where we need it ATM.  We can handle discrepancy if/when we run into it.

Reply:
Done.  Note that, when pypandoc is not installed, there is extra output from `setup.py --name` that messes with the substitution.  This is fixed by #99.